contracts/6/libs/BaseRelayRecipient.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier:MIT
+pragma solidity 0.7.6;
+
+import "../interfaces/IRelayRecipient.sol";
```

Comment:
why we created this new contract? is it because we had to change the solidity version and gsn doesn't have 0.7.6 support?

Reply:
Yes exactly checked several repos didn t find compatible with 0.7.6